### PR TITLE
SVG: Bring back tests for basic behaviors

### PR DESCRIPTION
**Pull request by mixonic**
A first pass at ensuring proper SVG support. There seems to be an outstanding issue with nesting html content inside a `foreignObject` at the least.

### CODE REVIEW

File: packages/node_modules/glimmer-runtime/lib/syntax/core.ts [hunk 1]
```diff
@@ -420,8 +421,13 @@ export class DynamicAttr extends AttributeSyntax {
...
   compile(compiler: CompileInto, env: Environment) {
-    compiler.append(new PutValueOpcode({ expression: this.value.compile(compiler, env) }));
-    compiler.append(new DynamicAttrOpcode(this));
+    let {namespace, value} = this;
+    compiler.append(new PutValueOpcode({ expression: value.compile(compiler, env) }));
+    if (namespace) {
+      compiler.append(new DynamicAttrNSOpcode(this));
+    } else {
+      compiler.append(new DynamicAttrOpcode(this));
+    }
```
Comment (chancancode) [inline packages/node_modules/glimmer-runtime/lib/syntax/core.ts]: Good catch! I guess that means we didn't have tests for this case before :fearful:

File: packages/node_modules/glimmer-runtime/lib/compiled/opcodes/dom.ts [hunk 2]
```diff
@@ -178,7 +178,11 @@ export class NamespacedAttribute implements ElementPatchOperation {
   }
 
   apply(dom: DOMHelper, element: Element, lastValue: string = null): any {
-    let { value: reference, namespace } = this;
+    let {
+      name,
+      value: reference,
+      namespace
+    } = this;
     let value = reference.value();
```
Comment (chancancode) [inline packages/node_modules/glimmer-runtime/lib/compiled/opcodes/dom.ts]: :fearful:

File: packages/node_modules/glimmer-runtime/tests/initial-render-test.ts [hunk 3]
```diff
@@ -845,103 +849,87 @@ test("error line numbers include multiple mustache lines", function() {
...
-//   let html = '<svg><path stroke="black" d="M 0 0 L 100 100"></path></svg>';
-//   let template = compile(html);
-//   let svgNode = render(template, {}).fragment.firstChild;
+module("Initial render (svg)");
 
-//   equal( svgNode.childNodes[0].namespaceURI, svgNamespace,
-//          "creates the path element with a namespace" );
-//   equalTokens(svgNode, html);
-// });
+test("Simple elements can have namespaced attributes", function() {
```
Comment (mixonic) [inline packages/node_modules/glimmer-runtime/tests/initial-render-test.ts]: Note to self: mmun sez these test names can be improved.

File: packages/node_modules/glimmer-runtime/lib/compiled/opcodes/dom.ts (hunk 2 above)
Comment (chancancode) [inline packages/node_modules/glimmer-runtime/lib/compiled/opcodes/dom.ts]: (Oops, this is one line off, I was again just surprised that the tests didn't catch the mistake!)

File: packages/node_modules/glimmer-runtime/tests/initial-render-test.ts (hunk 3 above)
Comment (mmun) [inline packages/node_modules/glimmer-runtime/tests/initial-render-test.ts]: Specifically, what is a "simple element" vs an "SVG element"?

**Pull request merged**